### ISSUE TITLE
Feature Idea: Helper to delete branches that exist only locally but not on the remote

### ISSUE THREAD

**Issue by bergmeister** [dahlbyk/posh-git#507]
VSTS offers the option of automatically deleting the remote branch upon completion of the PR, which is nice. However, this leaves me with local branches that I have to cleanup since getting a clean checkout is more expensive compared to that.
It would be nice to have a helper of some kind (a function, a git alias or whatever) to delete those leftover remote branches. Let me know what you think.

**Comment by rkeithhill** [comment]
I use this PowerShell command:
```posh
git branch --merged | where {$_ -match "^\s+user/$env:USERNAME/"} | foreach {git branch -d $_.trim()}
```
This is based on my team using a naming convention for private topic branches e.g. `user/<username>/<topic-branch-name>`.   But you could change that filter.  I believe another version of that checks for `'^*'` to make sure you don't delete the current branch.

**Comment by rkeithhill** [comment]
Is this a dupe of #79?

**Comment by dahlbyk** [comment]
> Is this a dupe of #79?

Yeah, closing as a duplicate. This is mostly implemented by #159, if someone wants to bring it up-to-date and address the remaining feedback.
